core/dataflow/src/main/java/edu/uci/ics/texera/dataflow/plangen/LogicalPlan.java

```diff
@@ -286,15 +287,14 @@ private void buildOperators() throws PlanGenException {
      *     there's no cycles in this DAG.
      *   each operator must meet its input and output arity constraints.
      *   the operator graph has at least one source operator.
-     *   the operator graph has exactly one sink.
+     *   the operator graph has at least one sink.
      * 
      * Throws PlanGenException if the operator graph is invalid.
      */
     private void validateOperatorGraph() throws PlanGenException {
         checkGraphConnectivity();
         checkGraphCyclicity();
         checkOperatorInputArity();
```

Comment:
Why is `checkOperatorOutputArity();` removed? If the new backend workflow design makes the tests in this function not passing, this function should be properly modified instead of just removing it.

Reply:
Yes, removing this function is a temporary solution. 
Actually, if we want to support mutiple sink operators, the `checkOperatorArity()` should only be applied to sink operators.

Reply:
Do you mean `checkOperatorOutputArity` when you say `checkOperatorArity`? I feel like we can still apply `checkOperatorOutputArity` to each operators, we can modify `checkOperatorOutputArity` to that it just make sure that each operator's output count is no less that the expected count. (if 1 output is expected, we check if actual output count is >= 1, something like this?)

Reply:
Yes, I will modify `checkOperatorOutputArity` method.